packages/nextjs/hooks/scaffold-stark/__tests__/useScaffoldMultiWriteContract.test.ts

```diff
@@ -0,0 +1,236 @@
+import { renderHook, act } from "@testing-library/react";
+
+import { vi, describe, it, expect, beforeEach, type Mock } from "vitest";
+
+import {
+  useScaffoldMultiWriteContract,
+  createContractCall,
+} from "../useScaffoldMultiWriteContract";
+
+import { useTargetNetwork } from "../useTargetNetwork";
+
+import { useNetwork, useSendTransaction } from "@starknet-react/core";
+
+import { useTransactor } from "../useTransactor";
+
+import { useDeployedContractInfo } from "~~/hooks/scaffold-stark";
+
+import { Contract, RpcProvider } from "starknet";
+
+// Mock the external dependencies
+
+vi.mock("~~/hooks/scaffold-stark/useTargetNetwork", () => ({
+  useTargetNetwork: vi.fn(),
+}));
+
+vi.mock("@starknet-react/core", () => ({
+  useSendTransaction: vi.fn(),
+
+  useNetwork: vi.fn().mockReturnValue({ chain: { id: 1 } }),
+}));
+
+vi.mock("starknet", () => {
+  const actualStarknet = vi.importActual("starknet");
+
+  return {
+    ...actualStarknet,
+
+    Contract: vi.fn(),
+
+    RpcProvider: vi.fn(),
+  };
+});
+
+vi.mock("../useTransactor");
+
+vi.mock("~~/hooks/scaffold-stark", () => ({
+  useDeployedContractInfo: vi.fn(),
+
+  useTransactor: vi.fn(),
+}));
+
+describe("useScaffoldMultiWriteContract Hook", () => {
+  const mockSendTransaction = vi.fn();
+
+  const mockTransactor = vi.fn((fn) => fn());
+
+  const mockedUseNetwork = useNetwork as unknown as Mock;
```

Comment:
You don't need `unknown` here, and you can move this to outside the `describe` block